### ISSUE TITLE
Workflow not triggered when merging pre-release branch

### ISSUE THREAD

**Issue by aurelien-reeves** [cucumber-actions/create-release#1]
Refs. https://github.com/cucumber/cucumber-ruby-wire/actions/workflows/release.yaml

Attempted to release cucumber-ruby-wire.

When creating the release branch as part of the pre-release process, the release workflow has been triggered. But as it did not yet contains the actual new release, it hopefully failed.

Then, when finally merging the changes from the pre-release PR, it has not been triggered. Don't know why. So the release had to be done manually.

Edit: rephrased

**Comment by mattwynne** [comment]
> On cucumber-ruby-wire, the release workflow has been triggered when creating the release branch

Yeah, we don't want this. I think we need to experiment with using a different event to trigger the release workflow.

**Comment by mattwynne** [comment]
See https://github.com/cucumber-actions/create-release-pr/issues/65#issuecomment-915434240
